pmm/pmm2-server-release.groovy

```diff
@@ -177,17 +181,24 @@ pipeline {
                         fi
                         docker save percona/pmm-server:\${VERSION} | xz > pmm-server-\${VERSION}.docker
 
-                        #docker pull \${DOCKER_CLIENT_VERSION}
-                        #docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
-                        #docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
-                        #docker push perconalab/pmm-client:\${VERSION}
-                        #docker save perconalab/pmm-client:\${VERSION} | xz > pmm-client-\${VERSION}.docker
+                        docker pull \${DOCKER_CLIENT_VERSION}
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${DOCKER_MID}
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${TOP_VER}
+                        if [ \${TOP_VER} = 1 ]; then
```

Comment:
@EvgeniyPatlan @hors shouldn't it be `2`?

Reply:
@Sevlin ^

Reply:
@AlekSi As I know we do not tag pmm 2 as latest for now.

Reply:
True. But we also do not build PMM 1 with that job. So why this check exist at all?